More Promisify for chrome API usage.

hidehikoabe opened the pull request:
This is preparation to introduce co library.
Use more promisified Chrome API in the extension.

issue #9

hidehikoabe committed:
More Promisify for chrome API usage.

This is preparation to introduce co library.
Use more promisified Chrome API in the extension.

nya3jp commented on src/permissions.js:
```diff
@@ -12,34 +12,58 @@
 };
 
+/**
+ * @param {Promise} api - Promisified chrome API. See also chromeasync module.
```
|api| is not actually a Promise but a function returning a Promise?

hidehikoabe replied:
Good catch. Fixed. Along with the change, annotated type parameter to the Promise, too.

nya3jp commented:
LGTM

hidehikoabe committed:
Update JSDoc.

Annotated the type of the result value to Promise.
Fix the name of "@param {Store}".
